docker-compose.yml

```
@@ -0,0 +1,9 @@
+services:
+  simple-jade-pin-server:
+    build:
+      context: .
```

```suggestion
    build: .
```
This can be abbreviated